src/languages/php.md

```diff
@@ -233,11 +231,10 @@ Another example is to set the timezone of the PHP runtime (though, the timezone
 date.timezone = "Europe/Paris"
 ```
 
-After pushing your file, you can check that the custom PHP configuration
-has been added to your environment by logging in via SSH and type:
+After pushing your file, you can check that the custom PHP configuration has been added to your environment by logging in via SSH and type:
```

Comment:
I don't know; that line has been in the docs forever. :smile: I think the intent is to make sure you didn't typo something.  The Redis docs have a similar stanza.  I'm open to removing it, though.